pkg/cache/server/config.go

```diff
@@ -148,7 +146,6 @@ func NewConfig(opts *cacheserveroptions.CompletedOptions, optionalLocalShardRest
 		return apiHandler
 	}
 
-	opts.Etcd.StorageConfig.Paging = utilfeature.DefaultFeatureGate.Enabled(features.APIListChunking)
```

Comment:
is this field gone or should it be true by default?

Reply:
The option is completely gone.

Reply:
Pagination is enforced since https://github.com/kubernetes/kubernetes/pull/121390 if I understand correctly.